libsrc/io_int_stubs.f90

```diff
@@ -0,0 +1,157 @@
+! Stubs version of wrf io spi subroutines
+!
+!--- get_dom_ti_real
```

Comment:
Who wrote this code?

Doxygen documentation is required.

Reply:
I took all this code straight from the GSI repository and just copied it over. I don't know who wrote it. I'll go through and bring it up to standard.